src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
+
+    public static void add(String fruit, int quantity) {
+        fruits.put(fruit, fruits.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public static void subtract(String fruit, int quantity) {
+        fruits.put(fruit, fruits.getOrDefault(fruit, 0) - quantity);
```

Comment:
The `subtract` method should check if the resulting quantity is negative before updating the map. If the quantity becomes negative, consider throwing a `RuntimeException` to prevent invalid states in the storage.